src/Views/ListView/Lists/MusicListView.vala

```diff
@@ -84,109 +82,73 @@ public class Noise.MusicListView : GenericList {
-    }
-
-    public void build_ui () {
-        button_release_event.connect(viewClickRelease);
-
-        mediaScrollToCurrent = new Gtk.MenuItem.with_label(_("Scroll to Current Song"));
-        mediaScrollToCurrent.sensitive = false;
-        mediaScrollToCurrentSeparator = new Gtk.SeparatorMenuItem ();
-        mediaTopSeparator = new Gtk.SeparatorMenuItem ();
-        mediaEditMedia = new Gtk.MenuItem.with_label(_("Edit Song Info"));
-        mediaFileBrowse = new Gtk.MenuItem.with_label(_("Show in File Browser"));
-        mediaMenuContractorEntry = new Gtk.MenuItem.with_label(_("Other actions"));
-        mediaMenuQueue = new Gtk.MenuItem.with_label(C_("Action item (verb)", "Queue"));
-        mediaMenuAddToPlaylist = new Gtk.MenuItem.with_label(_("Add to Playlist"));
-        mediaRemove = new Gtk.MenuItem.with_label(_("Remove Song"));
-        importToLibrary = new Gtk.MenuItem.with_label(_("Import to Library"));
-        mediaRateMedia = new Granite.Widgets.RatingMenuItem ();
-
-        mediaActionMenu = new Gtk.Menu ();
-        mediaActionMenu.attach_to_widget (this, null);
-
+        button_release_event.connect(view_click_release);
+
+        media_scroll_to_current = new Gtk.MenuItem.with_label(_("Scroll to Current Song"));
+        media_scroll_to_current.sensitive = false;
+        media_scroll_to_current.activate.connect(media_scroll_to_current_requested);
+        
+        media_edit_media = new Gtk.MenuItem.with_label(_("Edit Song Info"));
+        media_edit_media.activate.connect(media_menu_edit_clicked);
+        
+        media_file_browse = new Gtk.MenuItem.with_label(_("Show in File Browser"));
+        media_file_browse.activate.connect(media_file_browse_clicked);
+        
+        media_menu_contractor_entry = new Gtk.MenuItem.with_label(_("Other actions"));
+        
+        media_menu_queue = new Gtk.MenuItem.with_label(C_("Action item (verb)", "Queue"));
+        media_menu_queue.activate.connect(media_menu_queue_clicked);
+        
+        media_menu_add_to_playlist = new Gtk.MenuItem.with_label(_("Add to Playlist"));
+        
+        media_remove = new Gtk.MenuItem.with_label(_("Remove Song"));
+        media_remove.activate.connect(mediaRemoveClicked);
+        
+        import_to_library = new Gtk.MenuItem.with_label(_("Import to Library"));
+        import_to_library.activate.connect(import_to_library_clicked);
+        
+        media_rate_media = new Granite.Widgets.RatingMenuItem ();
+        media_rate_media.activate.connect(media_rate_media_clicked);
+
+        media_action_menu = new Gtk.Menu ();
+        media_action_menu.attach_to_widget (this, null);
         if(hint != ViewWrapper.Hint.ALBUM_LIST) {
-            //mediaActionMenu.append(browseSame);
-            mediaActionMenu.append(mediaScrollToCurrent);
-            mediaActionMenu.append(mediaScrollToCurrentSeparator);
+            //media_action_menu.append(browseSame);
+            media_action_menu.append(media_scroll_to_current);
```

Comment:
It would be perfect if you made sure every opening parenthesis is preceded by a space, but if you don't want to check the whole file, just tell and I'll make a little commit.